src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -720,6 +720,19 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     </EmbeddedResource>
   </ItemDefinitionGroup>
 
+  <!--
+    Target that allows targets consuming source control confirmation to establish a dependency
+    on the targets producing this information.
+
+    Any target that reads SourceRevisionId, PrivateRepositoryUrl, and other source control related
+    properties and items should depend on this target.
+
+    Source control information provider that sets these properties and items shall execute before
+    this target (by including InitializeSourceControlInformation in its BeforeTargets) and set
+    source control properties and items that haven't been initialized yet.
+  -->
+  <Target Name="InitializeSourceControlInformation" />
```

Comment:
Do we need to insert the DependsOn property we talked about somewhere so folks can inject into this but still have a package compatible with VS2015?

Reply:
Yes, i'll add the property. 

I also have one more piece of feedback coming from NuGet team to look into before we merge.